runtime/common/metering.go

```diff
@@ -235,6 +235,22 @@ func NewCompositeMemoryUsages(count uint64, elementSize uint) (MemoryUsage, Memo
 		}, leaves, branches
 }
 
+func NewAtreeMapPreAllocatedElementsMemoryUsage(size uint64) MemoryUsage {
+	leafNodesCount, _ := atreeNodes(size, 0)
+
+	const preAllocatedElements uint64 = 32
```

Comment:
Is there some constant in `atree` we can refer to for this value in case the underlying implementation changes in the future?